docs/source/tutorials/muon_GUI_course/the_tabs_results.rst

```diff
@@ -0,0 +1,89 @@
+to them in the table below. By default, checking the box next to a workspace in this table means its Log Values will be present in the results table. This can be changed with the Include/Exclude option
+(if Exclude is selected from the drop-down menu, checked workspaces will be the only ones *not* included in the table). The view can also be customised to only show selected workspaces.
+
+As an exercise, follow the instructions below in order to produce a results table for a single individual or sequential fit.
+
+1.  Load the HIFI00062798 file from the reference folder, guess alpha as described in :ref:`grouping <the_tabs_grouping>` then fit the ExpDecayOsc function to it. 
+    To instead demonstrate a sequential fit table, load the EMU00019631-4 files, don't guess alpha, and then perform a sequential fit of ExpDecayOsc on those files.
+    (See :ref:`using fit functions <data_fitting>` for instructions on single and sequential fits.)
+2.  In the Results tab, the default individual fit table should already be set up. Check that the Function Name and workspace(s) selected in the lower part of the tab show the 
+    fit function and data used so far, respectively.
+3.  Use the table in the 'Log Values' section to select parameters to include in the results table. This is done by checking the box next to them - try this now for 
+    run number and Temp_Sample.
+4.  Pick a name for the table, then click Output Results. See figure 33 for the process for an individual fit, and 34 for sequential.
+5. To view a table, right click it in the workspaces pane and Show Data.
+
+.. figure:: /images/thetabsresultsfig33.gif
+    :align: center
+    
+    Figure 33: How to create a results table from a single individual fit. 
+
+.. figure:: /images/thetabsresultsfig34.gif
+    :align: center
+
+    Figure 34: How to create a results table from a sequential fit. 
+
+Plotting from a Results Table
+-----------------------------
+
+Once a results table has been created, there are now different sets of parameters available for individual analysis. In Mantid, it is possible to plot different parameters 
+against each other, to see the relationship between the two. 
+
+Follow the instructions below in order to plot a graph from parameters in a Results Table. 
+
+1.  Files EMU00019631.nxs to EMU00019634.nxs should already have been loaded, sequentially fitted and a Results Table produced from them during the last section. 
+    If not then load the files, fit and produce a table. 
+2.  This example plots Temp_Sample against Lambda, which should automatically be assigned to the X and Y axes by Mantid (labelled X1 and Y1 respectively) click on Temp_Sample to select it. 
+        
+*NB If data is not automatically assigned to the desired axes this can be changed manually. As an example, if in step 2. Temp_Sample was not already assigned to X, 
```

Comment:
```suggestion
*NB If data is not automatically assigned to the desired axes this can be changed manually. As an example, if in step 2. `Temp_Sample` was not already assigned to X, 
```